tutorial_src/gue/doc/spiral.cls

```diff
@@ -1,6 +1,33 @@
 \NeedsTeXFormat{LaTeX2e}
 \ProvidesClass{spiral}
 
+\def\sinolShortNameText{Skr\'ot}
+\def\sinolRAMText{Limit pami\k eci}
+\def\sinolTimeText{Limit czasu}
+\def\sinolInputText{Wej\'scie dla testu}
+\def\sinolOutputText{Wyj\'scie dla testu}
+\def\sinolOtherSamplesText{Inne testy przyk\l adowe}
+\def\sinolSubtaskText{Podzadanie}
+\def\sinolPointsText{Punkty}
```

Comment:
To Skr\'ot to na pewno tak powinno być?

Reply:
Owszem, tak robi się w clsie polskie znaki. Możesz zobaczyć że PDFy dobrze się wygenerowały

Reply:
kk